PGM/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -471,6 +451,9 @@ private void renderSidebar() {
                 }
             }
 
+            rows.add("          ");
+            rows.add(ChatColor.AQUA + "play.stratus.network");
+
             // Need at least one row for the sidebar to show
             if(rows.isEmpty()) {
```

Comment:
at this current state of the code, this is redundant

i'm expecting a config for the above though, so it should stay